faraday/uart/layer_2_protocol.py

```diff
@@ -15,14 +15,20 @@
 import time
 import struct
 import logging
+import sys
 
 # Get configured logger
 logger = logging.getLogger('UARTStack')
 
 
 class layer_2_object(object):
```

Comment:
I seem to only use it locally in layer_2_object() class. I agree that this may be a bit convoluted and unneeded but the protocol object itself certainly is used. I believe it's best to leave it here for now and have a seperate IT/PR to update the lower level UART code for cleanliness/optimization. We're going to need to likely do this anyways with the improvement of generic data operation on the near term horizon.